hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/MiniOzoneOMHAClusterImpl.java

```diff
@@ -54,10 +54,10 @@
  * with OM HA suitable for running tests.  The cluster consists of a set of
  * OzoneManagers, StorageContainerManager and multiple DataNodes.
  */
-public class MiniOzoneHAClusterImpl extends MiniOzoneClusterImpl {
```

Comment:
Comments need to be updated witth MiniOzoneHAClusterImpl -> MiniOzoneOMHAClusterImpl